diff --git a/dist/vanilla-extract-rollup-plugin.cjs.dev.js b/dist/vanilla-extract-rollup-plugin.cjs.dev.js
index 6179c524c1f61657d95c36b13b4e588dadeae674..bbe36299a1b2fdf7a7722427a335b5471926245b 100644
--- a/dist/vanilla-extract-rollup-plugin.cjs.dev.js
+++ b/dist/vanilla-extract-rollup-plugin.cjs.dev.js
@@ -250,7 +250,8 @@ function vanillaExtractPlugin({
           name: sourcemapName,
           originalFileName: sourcemapName,
           source: bundle.generateMap({
-            file: name
+            file: name,
+            includeContent: true,
           }).toString()
         });
       }
diff --git a/dist/vanilla-extract-rollup-plugin.cjs.prod.js b/dist/vanilla-extract-rollup-plugin.cjs.prod.js
index f56cc872385986648aa5072b4572358df89482da..cd94d31eb22bce15b2ee1633fc8100fa183bed97 100644
--- a/dist/vanilla-extract-rollup-plugin.cjs.prod.js
+++ b/dist/vanilla-extract-rollup-plugin.cjs.prod.js
@@ -249,7 +249,8 @@ function vanillaExtractPlugin({
           name: sourcemapName,
           originalFileName: sourcemapName,
           source: bundle.generateMap({
-            file: name
+            file: name,
+            includeContent: true,
           }).toString()
         });
       }
diff --git a/dist/vanilla-extract-rollup-plugin.esm.js b/dist/vanilla-extract-rollup-plugin.esm.js
index 951c243d5c16e897742c9c2aa356ac2616c478f1..129589d20911fb799dccedfd0a1523fa57987181 100644
--- a/dist/vanilla-extract-rollup-plugin.esm.js
+++ b/dist/vanilla-extract-rollup-plugin.esm.js
@@ -242,7 +242,8 @@ function vanillaExtractPlugin({
           name: sourcemapName,
           originalFileName: sourcemapName,
           source: bundle.generateMap({
-            file: name
+            file: name,
+            includeContent: true,
           }).toString()
         });
       }
